src/templates/next/components/shared/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import { BiLeftArrowAlt, BiRightArrowAlt } from "react-icons/bi"
+import type { PaginationProps } from "../../../types/Pagination"
+
+export const Pagination = ({
+  totalItems,
+  itemsPerPage,
+  currPage,
+  setCurrPage,
+}: Omit<PaginationProps, "type">) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage)
+
+  return (
+    <nav className="flex flex-row w-full" aria-label="Pagination">
+      {/* Previous button */}
+      <button
+        className="flex flex-row p-1 gap-1 align-middle cursor-pointer disabled:cursor-not-allowed disabled:text-interaction-sub disabled:hover:bg-transparent hover:bg-interaction-main-subtle-hover"
+        aria-label="Previous page"
+        disabled={currPage === 1}
```

Comment:
i think the assumption that the starting index is 1 is pretty important to make clear! 

not sure where the caller is but yea

Reply:
Yeah I was being explicit here with using page as the terminology to indicate that it should start with 1.